src/inspect.rs

```diff
@@ -1,18 +1,161 @@
+            annotations.remove(&String::from(*annotation));
+        }
+    }
+    table.add_row(row![Fgbl -> "mutating:", metadata.mutating]);
+    table.add_row(row![Fgbl -> "protocol version:", protocol_version]);
+
+    let _usage = annotations.remove("io.kubewarden.policy.usage");
+    if !annotations.is_empty() {
+        table.add_row(row![]);
+        table.add_row(row![Fmbl -> "Annotations"]);
+        for (annotation, value) in annotations.iter() {
+            table.add_row(row![Fgbl -> annotation, d -> value]);
+        }
+    }
+
+    let labels = metadata.labels.clone().unwrap_or_default();
+    if !labels.is_empty() {
+        table.add_row(row![]);
+        table.add_row(row![Fmbl -> "Labels"]);
+        for (label, value) in labels.iter() {
+            table.add_row(row![Fgbl -> label, d -> value]);
+        }
+    }
+    table.printstd();
+
+    Ok(())
+}
+
+fn print_metadata_rules(metadata: &Metadata) -> Result<()> {
+    let rules_yaml = serde_yaml::to_string(&metadata.rules)?;
+
+    // Quick hack to print a colorized "Rules" section, with the same
+    // style as the other sections we print
+    let mut table = Table::new();
+    table.set_format(FormatBuilder::new().padding(0, 1).build());
+    table.add_row(row![Fmbl -> "Rules"]);
+    table.printstd();
+
+    let text = format!("```yaml\n{}```", rules_yaml);
+    render_markdown(&text)
+}
+
+fn print_metadata_usage(metadata: &Metadata) -> Result<()> {
+    let usage = match metadata.annotations.clone() {
+        None => None,
+        Some(annotations) => match annotations.get("io.kubewarden.policy.usage") {
+            Some(usage) => Some(String::from(usage)),
+            None => None,
+        },
+    };
+
+    if usage.is_none() {
+        return Ok(());
+    }
+
+    // Quick hack to print a colorized "Rules" section, with the same
+    // style as the other sections we print
+    let mut table = Table::new();
+    table.set_format(FormatBuilder::new().padding(0, 1).build());
+    table.add_row(row![Fmbl -> "Usage"]);
+    table.printstd();
+
+    render_markdown(&usage.unwrap())
+}
+fn render_markdown(text: &String) -> Result<()> {
+    let size = TerminalSize::detect().unwrap_or_default();
+    let columns = size.columns;
+    let settings = mdcat::Settings {
+        terminal_capabilities: TerminalCapabilities::detect(),
+        terminal_size: TerminalSize { columns, ..size },
+        resource_access: ResourceAccess::LocalOnly,
+        syntax_set: SyntaxSet::load_defaults_newlines(),
+    };
+    let parser = Parser::new_ext(
+        text,
+        Options::ENABLE_TASKLISTS | Options::ENABLE_STRIKETHROUGH,
+    );
+    let env = mdcat::Environment::for_local_directory(&std::env::current_dir()?)?;
+
+    let stdout = std::io::stdout();
+    let mut output = stdout.lock();
+    mdcat::push_tty(&settings, &env, &mut output, parser).or_else(|error| {
+        if error.kind() == std::io::ErrorKind::BrokenPipe {
```

Comment:
How can this happen?

Reply:
I have to say, this is coming from the mdcat source code. I think it could happen if `kwctl inspect` is piped into somethink like `less`